docs/shared/tutorials/gradle.md

```diff
@@ -0,0 +1,329 @@
+
+Project name (default: gradle-tutorial): gradle-tutorial
+
+Select application structure:
+  1: Single application project
+  2: Application and library project
+Enter selection (default: Single application project) [1..2] 2
+
+Generate build using new APIs and behavior (some features may change in the next minor release)? (default: no) [yes, no] no
+
+
+> Task :init
+To learn more about Gradle by exploring our Samples at https://docs.gradle.org/8.7/samples/sample_building_java_applications_multi_project.html
+
+BUILD SUCCESSFUL in 23s
+1 actionable task: 1 executed
+```
+
+Finally, set up git and create your first commit:
+
+```shell
+git init
+git commit -am "initial commit"
+```
+
+You can run the `gradle projects` command to get a list of projects in the repo.
+
+```text {% command="gradle projects" %}
+> Task :projects
+
+------------------------------------------------------------
+Root project 'gradle-tutorial'
+------------------------------------------------------------
+
+Root project 'gradle-tutorial'
++--- Project ':app'
++--- Project ':list'
+\--- Project ':utilities'
+
+To see a list of the tasks of a project, run gradle <project-path>:tasks
+For example, try running gradle :app:tasks
+
+BUILD SUCCESSFUL in 15s
+11 actionable tasks: 11 executed
+```
+
+The repository has two Java libraries (under `list` and `utilities`) that are used in an application located in the `app` folder.
+
+You can build the `app` project by running `gradle app:build`:
+
+```text {% command="gradle app:build" %}
+BUILD SUCCESSFUL in 3s
+21 actionable tasks: 12 executed, 9 up-to-date
+```
+
+## Add Nx
+
+With the `@nx/gradle` plugin, Nx can be easily added to a Gradle repository and Nx will work seamlessly along side Gradle.
+
+Nx offers many features, but at its core, it is a task runner. Out of the box, it can cache your tasks and ensure those tasks are run in the correct order. After the initial set up, you can incrementally add on other features that would be helpful in your organization.
+
+To enable Nx in your repository, run a single command:
+
+```shell {% path="~/gradle-tutorial" %}
+npx nx@latest init
+```
+
+This command will download the latest version of Nx and help set up your repository to take advantage of it.
+
+The script will propose installing the `@nx/gradle` plugin to allow Nx to understand Gradle configuration files. Make sure to select the plugin to install it.
+
+After Nx has been installed, there will be a few new files in your repository:
+
+```treeview
+gradle-tutorial/
+├── .nx/
+├── app/
+├── list/
+├── utilities/
+├── gradlew
+├── gradlew.bat
+├── nx
+├── nx.bat
+├── nx.json
+└── settings.gradle.kts
+```
+
+Because this is not a javascript repository, there is no `package.json` file or `node_modules` folder. Instead, Nx has provided executables at the root of the repository: `nx` for Unix operating systems and `nx.bat` for Windows. These executables can be used in the same way that `gradlew` and `gradlew.bat` are used.
```

Comment:
```suggestion
Because this is not a JavaScript repository, there is no `package.json` file or `node_modules` folder. Instead, Nx has provided executables at the root of the repository: `nx` for Unix operating systems and `nx.bat` for Windows. These executables can be used in the same way that `gradlew` and `gradlew.bat` are used.
```